WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Carriers and Rates/ShippingLabelCarrierRowViewModel.swift

```diff
@@ -47,7 +47,7 @@ struct ShippingLabelCarrierRowViewModel: Identifiable {
 
         title = rate.title
         let formatString = rate.deliveryDays == 1 ? Localization.businessDaySingular : Localization.businessDaysPlural
-        subtitle = String(format: formatString, rate.deliveryDays)
+        subtitle = String(format: formatString, rate.deliveryDays?.description ?? "-")
```

Comment:
This works, but what do you think about checking if `rate.deliveryDays` is null and, in that case, hiding the subtitle in the `ShippingLabelCarrierRow` view entirely? It looks like that's what we do in wp-admin:

![Screen Shot 2021-06-29 at 3 55 36 PM](https://user-images.githubusercontent.com/8658164/123820954-e6280600-d8f2-11eb-8d07-e629ada1e852.png)

Otherwise, we end up with "- business days" which is a little unclear:

<img src="https://user-images.githubusercontent.com/8658164/123821039-f8a23f80-d8f2-11eb-9de2-610f41e2e70f.png" width="300px">